apps/taxes/src/modules/avatax/maps/address-map.ts

```diff
@@ -0,0 +1,39 @@
+import { AddressLocationInfo as AvataxAddress } from "avatax/lib/models/AddressLocationInfo";
+import { ChannelAddress } from "../../channels-configuration/channels-config";
+
+export type SaleorAddress = {
```

Comment:
Is it https://docs.saleor.io/docs/3.x/api-reference/users/objects/address?

If yes, why not use generated fragment?

Reply:
done